ws-nextjs-app/pages/[service]/send/[id]/FormField/index.tsx

```diff
@@ -42,15 +111,43 @@ const FormComponents: Record<
   file: File,
 };
 
-const FormField = ({ id, htmlType, label }: FormComponentProps) => {
-  const Component = FormComponents[htmlType];
+export type FormComponentProps = {
+  id: string;
+  htmlType: HtmlType;
+  label: string;
+  type?: string;
+  textArea?: boolean;
+};
+
+const FormField = ({
+  id,
+  type,
+  htmlType,
+  label,
+  textArea,
+}: FormComponentProps) => {
+  // TODO: Don't like this but needed atm since 'file' and 'textarea' aren't returned as 'htmlType' from the API
+  // should probably do this in back-end
+  let derivedHtmlType = htmlType;
+
+  if (textArea) {
+    derivedHtmlType = 'textarea';
+  }
+
+  if (type === 'file') {
+    derivedHtmlType = 'file';
+  }
+
+  const Component = FormComponents[derivedHtmlType];
 
   if (!Component) return null;
 
   return (
     <div css={styles.formField}>
-      <Label id={id}>{label}</Label>
-      <Component id={id} name={label} />
+      <Label id={id}>
+        {label}
+        <Component id={id} name={id} />
```

Comment:
Looking at the existing Uploader, the `id` and `name` html attributes are the same value which is why I changed this.